config/features.yml

```
@@ -117,6 +117,10 @@ features:
     actor_type: user
     description: Enables language translation for the pre-check-in experience
     enable_in_development: true
+  check_in_experience_larota_security_updates_enabled:
```

minor typo, which we should fix to avoid confusion:
```suggestion
  check_in_experience_lorota_security_updates_enabled:
```

---

Thanks @randomsync I'll fix!